build/ci/github_actions/operator/save_driver_pods_logs.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -x
+
+get_all_pods_by_type (){
+    pod_type=$1
+    kubectl get pod -l csi | grep $pod_type | awk '{print$1}'
+}
+
+run_action_on_pod (){
+    pod_type=$1
+    action=$2
+    extra_args=$3
+    kubectl $action $(get_all_pods_by_type $pod_type) $extra_args > "/tmp/driver_$(get_all_pods_by_type $pod_type)_${action}.txt"
```

Comment:
does this mean we run `$(get_all_pods_by_type $pod_type)` twice for every call? do we need to?